### DISCUSSION TITLE
How I am supposed to return the view in the controller?

### DISCUSSION THREAD

**Discussion by perisicnikola37** [enlightn/enlightn#142] [Q&A] [answered]
Hello there,
I am returning the view page from my controller like this:
`public function index() {
return view('pages.books');
}
`

Enlightn gives me a reliability warning like this: "Method App\Http\Controllers\BookController::index() should return Illuminate\Http\Response but returns Illuminate\View\View."
How I am supposed to return the view in the controller to remove this warning? Thanks 😃

**Comment by paras-malhotra** [comment]
I think you need to just fix the docblock and add View as a return type hint

**Reply by paras-malhotra** [reply]
The docblock of your index method

**Comment by perisicnikola37** [comment, marked as answer]
All you need to do is make the method look like this where you return the view as a response.
Add this line of code before your method:
`* @return \Illuminate\View\View`
